README.md

```diff
@@ -76,9 +76,9 @@ The installer will also set Apache group ownership of certain /data/ subdirector
 
   To help ensure Apache-writability, verify that your environment file contains the following line:
 
-    ```bash
-    umask 0002
-    ```
+   ```bash
+   umask 0002
+   ```
 
```

Comment:
Minor detail, but I am not sure why you removed the spacing? it was consistent with the rest the way @christinerogers had it before.